### ISSUE TITLE
Multiple File Field Data Stored Incorrectly When Using WPML

### ISSUE THREAD

**Issue by Rig820** [pods-framework/pods#7380]
TLDR: File field data saved incorrectly when using PODs and WPML

I've used PODs to extend the WooCommerce product post type. The extension includes a Plain Paragraph Text field, a Plain Text field, and two File/Image/Video fields. Both File fields (downloads and certification_icons) are set to allow Multiple Files, and these are the two fields giving me an issue. The other two have had no issues at all. 
The other important piece of information is that I am using WPML on this site, with 3 languages. 

First noticed the issue when at different times the File data would not appear on the front end. Both fields were using a PODs template with an each function to output the data. Everything was fine on dev but after launching the site it was noticed that the data was missing on the front end. The odd part was that it could be missing for one language, two languages, all three languages, and any combination of them. If the data wasn't showing for one product in a language it wouldn't be showing for all products in that language. 
For example the field may be showing in French for all products but not in the other two languages. Clicking update on any post would 'reset' this. It may make the field show in all three languages again, but it was equally likely to make it disappear from all three, or make it work in just two of them. Any combination of the three could happen. 

Pods, Pod Groups, and Pod Fields are not set to be translatable. Neither are the two fields. 
![Image](https://github.com/user-attachments/assets/d533a390-ec2c-475e-8b4a-a53b52234156)
![Image](https://github.com/user-attachments/assets/ecfb572a-5191-41cc-a67a-ecd39cf5ee90)
![Image](https://github.com/user-attachments/assets/51785d63-06fc-47a3-bd3c-0ebd18b6a549)

The backend also wasn't showing correctly
 ![Image](https://github.com/user-attachments/assets/ee057423-1201-401b-8c13-1c28cf82ccaf)

This lead me to look at the data in the wp_postmeta table. The first few products I looked at the data looked fine, like below
![Image](https://github.com/user-attachments/assets/42431006-6abf-4115-86ed-77bf8d0b5315)

I then came across a product where instead of having a single media post id in each of the "download" and "certification_icons" rows it would contain an object
![Image](https://github.com/user-attachments/assets/bac47a30-afea-4f00-a377-d839ea320740)

If I manually fixed those rows to contain just the files post id it would then show correctly in the backend. The below images are when I just fixed the certification_icon rows
![Image](https://github.com/user-attachments/assets/cf632a06-8e39-48d4-bfd9-1da6e480f585)
![Image](https://github.com/user-attachments/assets/aa5c1ea6-d591-40a5-b28b-5b2b28e0ab84)

The problem is clicking on update after that would revert everything back to objects. I tried this with just fixing certification_icons, just downloads, and both. 

Finally if I deactivate WPML the files will display fine in the backend even though the data in the database is still objects. 
![Image](https://github.com/user-attachments/assets/64a2f73b-0b8f-48fd-bd00-14288d29741d)

Updating the product, no actual changes need be made,  while WPML is deactivated will then save the data correctly as post ids. 
![Image](https://github.com/user-attachments/assets/daa9b4ca-90c6-4a9d-898b-72730bcae564)

Activating WPML and saving the product again, breaks it again
